README.md

```diff
@@ -2,8 +2,10 @@
 Plugin integrating [Dolby.io Communications](https://dolby.io) with the Unreal Engine.
 
 ## Supported environments
-- Unreal Engine 4.27.2 and 5.0.3
-- Windows 10 and macOS 12
+- Windows 10+
```

Comment:
I miss some info about UE versions for Windows e.g. `(all EU versions)`
```suggestion
- Windows 10+ (all UE versions)
```

Reply:
I think this should be worded differently. And it is certainly not "all UE versions" ;). Maybe something like in https://api-references.dolby.io/comms-sdk-cpp/other/supported_platforms.html

```
## Supported platforms
The plugin is compatible with the following Unreal Engine versions:
- 4.27.2
- 5.0.3

The plugin is compatible with the following operating systems:
- Windows 10, Windows 11
- macOS:
    - UE4: latest Mojave or newer
    - UE5: latest Monterey or newer
```